### PR TITLE
[NodeManagerMapper]The sql annotation form is optimized to XML

### PR DESCRIPTION
**Pull request by binbinCheng**
[NodeManagerMapper]The sql annotation form is optimized to XML

Related issues: #3653

### CODE REVIEW

File: linkis-computation-governance/linkis-manager/linkis-manager-persistence/src/main/java/org/apache/linkis/manager/dao/impl/NodeManagerMapper.xml [hunk 1]
```diff
@@ -29,7 +29,82 @@
         update linkis_cg_manager_label_service_instance set service_instance = #{instance} where service_instance = #{tickedId}
     </update>
 
+    <insert id="addNodeInstance" useGeneratedKeys="true" keyColumn="id" keyProperty="id"
+            parameterType="org.apache.linkis.manager.common.entity.persistence.PersistenceNode">
+        insert into linkis_cg_manager_service_instance(instance,name,owner,mark,update_time,create_time,updator,creator)
+        values(#{instance},#{name},#{owner},#{mark},#{updateTime},#{createTime},#{updator},#{creator})
+    </insert>
 
+    <update id="updateNodeInstance">
```
Comment (casionone) [inline linkis-computation-governance/linkis-manager/linkis-manager-persistence/src/main/java/org/apache/linkis/manager/dao/impl/NodeManagerMapper.xml:38]: add parameterType

**Review by jacktao007 — APPROVED** [review]
LGTM.

File: linkis-computation-governance/linkis-manager/linkis-manager-persistence/src/main/java/org/apache/linkis/manager/dao/impl/NodeManagerMapper.xml [hunk 2]
```diff
@@ -29,7 +29,82 @@
...
+        values(#{engineNodeInstance}, #{emNodeInstance}, now(), now())
+    </insert>
+    <delete id="deleteEngineNode">
+        delete from  linkis_cg_manager_engine_em where engine_instance = #{engineNodeInstance} and em_instance = #{emNodeInstance}
+    </delete>
+    <select id="getNodeInstanceIdsByOwner" resultType="java.lang.Integer">
+        select id from linkis_cg_manager_service_instance where owner = #{owner}
+    </select>
 
 </mapper>
```
Comment (jacktao007) [inline linkis-computation-governance/linkis-manager/linkis-manager-persistence/src/main/java/org/apache/linkis/manager/dao/impl/NodeManagerMapper.xml:110]: no new line at end of file

**Review by jacktao007 — CHANGES_REQUESTED** [review]

**Review by casionone — DISMISSED** [review]

**Review by jacktao007 — APPROVED** [review]
LGTM.

**Pull request merged**